### DISCUSSION TITLE
Method scope issue

### DISCUSSION THREAD

**Discussion by pratikshap31** [enlightn/enlightn#121] [General]
Check 68/128: Your application does not contain invalid method calls. Failed
Your application seems to contain invalid method calls to methods that either do not exist or do not match the method signature or scope.

how to solve this issue?
Cannot call method format() on Carbon\Carbon|false.

I have added mixed responses but it is not working.

**Comment by paras-malhotra** [comment]
Can you share the code snippet that's triggering the issue?

**Comment by paras-malhotra** [comment]
The `createFromFormat` method of Carbon either returns a `Carbon` instance or false as it shows in the docblock here: https://github.com/briannesbitt/Carbon/blob/87a5e6f250b00f4422b2859baf60fb2c5d543e97/src/Carbon/Carbon.php#L505.

To fix this, you can do any of these:
1. Check for a falsy value before calling format
2. You can [ignore the error](https://www.laravel-enlightn.com/docs/getting-started/configuration.html#ignoring-errors) or [establish a baseline](https://www.laravel-enlightn.com/docs/getting-started/usage.html#establishing-a-baseline)
3. Use [PHPStan annotations](https://phpstan.org/user-guide/ignoring-errors) to ignore the errors

**Reply by pratikshap31** [reply]
thank you @paras-malhotra 

It is working.
